ckan/templates/user/snippets/api_token_list.html

```diff
@@ -38,7 +38,7 @@
               <form action="{{ action }}" method="POST">
                 {{ h.csrf_input() }}
                 <div class="btn-group">
-                  <button type="submit" href="{{ action }}" class="btn btn-danger btn-sm" title="{{ _('Revoke') }}" data-module="confirm-action">
+                  <button type="submit" href="{{ action }}" class="btn btn-danger btn-sm" aria-label="{{ _('Revoke ApiToken') }}" data-bs-title="{{ _('Revoke ApiToken') }}" data-bs-toggle="tooltip" data-module="confirm-action">
```

Comment:
The term `ApiToken` is used in code. In normal language, you should not use random case swaps **inSiDe the WoRd**. 
Check the heading of the table: we are using `token` in UI when referring to this thing.